assignment/src/com/assignmentweek2/person/Doctor.java

```diff
@@ -19,23 +19,15 @@ public String getAddress() {
     public void setAddress(String address) {
         this.address = address;
     }
+
+
     @Override
-    void Studentdisplay() {
-    }
-    @Override
-    void Teacherdisplay() {
-    }
-    void Doctordisplay(){
+    void display() {
         System.out.println("Doctor Role: "+getName());
     }
+
     @Override
-    void StudentAddress() {
-    }
-    @Override
-    void TeacherAddress() {
-    }
-    @Override
-    void DoctorAddress() {
+    void Address() {
```

Comment:
method name should start with small letter.

Reply:
done Nagarjuna